plaso/cli/extraction_tool.py

```diff
@@ -123,6 +127,24 @@ def _CreateProcessingConfiguration(self, knowledge_base):
 
     return configuration
 
+  def _GetParserPresetsInformation(self):
```

Comment:
Please move this functionality to the ParserManager. There's already similar methods (GetParsersInformation) there for producing lists of tuples like this for parsers and plugins.

Reply:
ack